src/components/SideBar/SideBar.module.css

```diff
@@ -84,3 +84,17 @@
 .DisabledLink {
   pointer-events: none;
 }
+
+.SideFooter {
+  height: 30px;
+  font-size: 12px;
+  line-height: 14px;
+  bottom: 0;
+  color: #008ac1;
+  padding: 18px 0 20px 18px;
+  cursor: pointer;
+  background-color: #06222f;
+}
+.SideFooter:hover {
+  color: rgb(131, 182, 202);
```

Comment:
still have issues with `rgb` if we don't need an alpha channel (transparency). let's stick with `hex`